src/pdfnorm.jl

```diff
@@ -0,0 +1,50 @@
+"""
+    pdfL2norm(d::Distribution)
+
+Return the L2 norm of the probability distribution function `f(x)` of the distribution `d`:
```

Comment:
```suggestion
Return the L2 norm of the probability density function `f(x)` of the distribution `d`:
```
It would be good to include `pdfL2norm` also in the documentation. I guess, it belongs to https://juliastats.org/Distributions.jl/latest/univariate/#Computation-of-statistics even though `pdf` is listed in https://juliastats.org/Distributions.jl/latest/univariate/#Probability-Evaluation.